sklearn/tree/tree.py

```diff
@@ -782,9 +783,10 @@ class DecisionTreeRegressor(BaseDecisionTree, RegressorMixin):
     Parameters
     ----------
     criterion : string, optional (default="mse")
-        The function to measure the quality of a split. The only supported
-        criterion is "mse" for the mean squared error, which is equal to
-        variance reduction as feature selection criterion.
+        The function to measure the quality of a split. Supported
+        criterions are "mse" for the mean squared error, which is 
+        equal to variance reduction as feature selection criterion, 
+        and "mae" for the mean absolute deviation.
```

Comment:
Docstrings for ExtraTreeRegressor should also be updated, along with the docstrings for the forests.